.travis.yml

```diff
@@ -19,7 +19,10 @@ before_install:
   - sudo apt-get install -y xz-utils lftp
 
 install:
-  - git clone --depth 1 https://github.com/thesofproject/kconfig.git
+  - git clone --depth 5 https://github.com/thesofproject/kconfig.git
+  - pushd kconfig
+  - &short_log git --no-pager log --oneline --graph --decorate --max-count=5
```

Comment:
why do we care about the 5 last commits, the last one is enough to uniquely tag the version? And why 5?

Reply:
5 commits provides more context so it's much faster to verify the current location by merely looking at the commit subjects, not even using the commit SHA1s.

Some commits have an "anonymous" /  useless subject,  examples from this kconfig repo[*]:
```
0ff799e73806 Update the README
caee716a24ff README.md: update doc
79e24692eebf base-defconfig: updates
e33ddf5404fb update README.md
```

From [past experience backporting thousands of commits](https://chromium-review.googlesource.com/q/marc.herbert%2540intel.com,75), the chances of having 5 consecutive _and_ meaningless subjects is very low.

There is even more value have more than one commit for the linux kernel repo at line 39. In that case the SHA1s are "volatile"/temporary because they haven't been merged yet. So if a subject happens to be meaningless then you can't even fallback on manually searching the SHA1.

[*] `git log --oneline | grep -i update`; easy.